src/com/facebook/buck/apple/project_generator/ProjectGenerator.java

```diff
@@ -2720,13 +2720,17 @@ ProductType bundleToTargetProductType(
           switch (extension) {
```

Comment:
It's weird to have a switch statement with only one case to begin with, change it to an `else if` instead.